treeherder/log_parser/tasks.py

```diff
@@ -88,8 +90,19 @@ def crossreference_error_lines(job_id, priority):
     newrelic.agent.add_custom_parameter("job_id", job_id)
     logger.debug("Running crossreference-error-lines for job %s" % job_id)
     job = Job.objects.get(id=job_id)
-    has_lines = crossreference_job(job)
+    try:
+        has_lines = crossreference_job(job)
+    except Exception as e:
+        import traceback
+        logger.error(traceback.format_exception(e))
```

Comment:
Wouldn't `logger.exception()` give the same result, but without the need to use `traceback` manually?

That said, do we need this? We don't do any such special handling anywhere else.